engines/python/src/test/resources/echo/model.py

```diff
@@ -39,6 +42,8 @@ def handle(inputs: Input):
         return "invalid_type"
     elif inputs.contains_key("exit"):
         sys.exit()
+    elif inputs.contains_key("OOM"):
+        raise OutOfMemoryError
```

Comment:
You can just use built-in exception MemoryError

Reply:
pytorch throws `OutOfMemoryError`, so thought it makes more sense to test that particular case.

Reply:
This is mock test anyway. No difference between built-in exception vs customized class

Reply:
Updated to keep it clean. Although there's a minor difference, when code change happens for some reason where we check for `OutOfMemoryError`, this test catches incorrect changes.